examples/functions/error/ErrorResponse_1.xml

```diff
@@ -15,7 +15,6 @@
 						<ojp:OJPGenericProblemType>OJPGENERIC_DATAFRAMEREFNOTAVAILABLE</ojp:OJPGenericProblemType>
 					</ojp:OJPError>
 				</siri:ErrorCondition>
-				<!-- no CalcTime for this service <ojp:CalcTime>20</ojp:CalcTime> -->
```

Comment:
@ue71603 please preview?

Reply:
@skinkie  Do you think I should add CalcTime to the service?

Reply:
Or remove the comment.

Reply:
It is removed....Isn't it?

Reply:
that's what I did...